doc/VMSpecificationSingleNode

```diff
@@ -0,0 +1,11 @@
+### VM Specifications - Single-Node VM deployment
+
+- Please create VM(s) with at least 1 Node, 2 cores (vCPU) and 4GB of RAM.
```

Comment:
I did not get `at least 1 Node` here. I guess all specifications are for Single Node V only. 

Also we should validate steps with mentioned specification. I used SSC VM's with higher capacity as part of developing steps and validating docker images

Reply:
done